src/Service/OrderService.php

```diff
@@ -22,7 +22,6 @@ class OrderService
      */
     public function __construct(private QueueInterface $queue)
     {
-        $this->queue = $queue;
         $this->eventSerializer = new EventSerializer();
```

Comment:
Might also property promote eventSerializer for consistency now we are at it.

Reply:
Promoted EventSerializer property.